build/commands/lib/config.js

```diff
@@ -210,6 +211,7 @@ Config.prototype.buildArgs = function () {
     enable_cdm_host_verification: this.enableCDMHostVerification(),
     skip_signing: !this.shouldSign(),
     chrome_pgo_phase: this.chromePgoPhase,
+    use_blink_v8_binding_new_idl_interface: this.useBlinkV8BindingNewIdlInterface,
```

Comment:
nit: I would just set this to false directly, there's no need for https://github.com/brave/brave-core/pull/6195/commits/3106679fc8b70ade590e068d62dae3697fb0706c#diff-40870b6e2fc6551484e69871b73f48e3R101

Reply:
Addressed by 4b5c88cac9f3170aa0b3691e413788325f7c57db